packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -528,9 +573,10 @@ async function handler(
         const end = req.body.allRecurringDates[i].end;
         if (isTeamEvent) {
           // each fixed user must be available
-          for (const key in fixedUsers) {
+          const fixedParticipants = [...fixedUsers, ...guestsList];
+          for (const key in fixedParticipants) {
             await ensureAvailableUsers(
-              { ...eventTypeWithUsers, users: [fixedUsers[key]] },
+              { ...eventTypeWithUsers, users: [fixedParticipants[key]] },
```

Comment:
We can't really reschedule all recurring events all together, I don't think so we need to add guestsList here.

cc @CarinaWolli

Reply:
Removed.